setup.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+with open('requirements.txt', 'r') as f:
```

Comment:
this is so I could do `python setup.py develop` and run stuff like `python flaskapp/models/__init__.py`